FusionAuth Partner Program Announcement (#3496)
FusionAuth/fusionauth-site: astro/src/content/blog/fusionauth-passwordless copy.mdx

- **DoiT**: Known for its expertise in cloud optimization and support, [DoiT](https://www.doit.com/) works with organizations to maximize the value of their cloud investments. Our collaboration ensures that FusionAuth installations can be optimized for performance and cost-efficiency in the cloud.

The FusionAuth Partner Program is designed to foster growth, collaboration, and innovation by offering:

- **Profitability**: Upfront and recurring discounts, value-added services, and promotional incentives to drive revenue growth.
- **Sales & Marketing Support**: Joint sales efforts, training, co-branded campaigns, and event participation to boost visibility and lead generation.
- **Technical Resources**: Access to a partner portal, training, and ongoing support to ensure successful implementations.
- **Growth Opportunities**: Tiered rewards, deal registration benefits, and collaboration to expand your business and market reach.

> FusionAuthWe believes that partnerships are vital to delivering the best possible solutions for our customers. The FusionAuth Partner Program is a significant step forward in achieving theour mission of making authentication and identity management easy, flexible, and secure for developers everywhere.

Reviewer: their or our?

> FusionAuthWe believes that partnerships are vital to delivering the best possible solutions for our customers. The FusionAuth Partner Program is a significant step forward in achieving theour mission of making authentication and identity management easy, flexible, and secure for developers everywhere.

Reviewer: At FusionAuth, we  or Fusion Auth believes

> FusionAuthWe believes that partnerships are vital to delivering the best possible solutions for our customers. The FusionAuth Partner Program is a significant step forward in achieving theour mission of making authentication and identity management easy, flexible, and secure for developers everywhere.

Reviewer: ```suggestion
We at FusionAuth believe that partnerships are vital to delivering the best possible solutions for our customers. The FusionAuth Partner Program is a significant step forward in achieving our mission of making authentication and identity management easy, flexible, and secure for developers everywhere.
```
